### PR TITLE
Moved LocalMapStatsImpl probes to fields

### PR DESCRIPTION
**Pull request by pveentjer**
Probe on a method is expensive due to litter. On a field
no litter is created.

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalMapStatsImpl.java [hunk 1]
```diff
@@ -61,12 +61,12 @@
             newUpdater(LocalMapStatsImpl.class, "removeCount");
 
     // The resolution is in nano seconds for the following latencies
-    private static final AtomicLongFieldUpdater<LocalMapStatsImpl> TOTAL_GET_LATENCIES =
-            newUpdater(LocalMapStatsImpl.class, "totalGetLatenciesNanos");
```
Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalMapStatsImpl.java]: Also renamed latencies to latency.

File: hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalMapStatsImpl.java [hunk 2]
```diff
@@ -450,9 +450,9 @@ public JsonObject toJson() {
         root.add("dirtyEntryCount", dirtyEntryCount);
 
         // keep the contract as milliseconds for latencies sent using Json
-        root.add("totalGetLatencies", NANOSECONDS.toMillis(totalGetLatenciesNanos));
-        root.add("totalPutLatencies", NANOSECONDS.toMillis(totalPutLatenciesNanos));
-        root.add("totalRemoveLatencies", NANOSECONDS.toMillis(totalRemoveLatenciesNanos));
+        root.add("totalGetLatencies", NANOSECONDS.toMillis(totalGetLatencyNanos));
```
Comment (pveentjer) [inline hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalMapStatsImpl.java]: The names for serializing to json have not been changed due to not breaking anything in MC. But this code will be removed anyway once we move to metrics for sending this data to MC.

**Comment by pveentjer** [comment]
run-lab-run

File: hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalMapStatsImpl.java [hunk 3]
```diff
@@ -539,9 +539,9 @@ public String toString() {
                 + ", getCount=" + getCount
                 + ", putCount=" + putCount
                 + ", removeCount=" + removeCount
-                + ", totalGetLatencies=" + NANOSECONDS.toMillis(totalGetLatenciesNanos)
-                + ", totalPutLatencies=" + NANOSECONDS.toMillis(totalPutLatenciesNanos)
-                + ", totalRemoveLatencies=" + NANOSECONDS.toMillis(totalRemoveLatenciesNanos)
+                + ", totalGetLatencies=" + NANOSECONDS.toMillis(totalGetLatencyNanos)
```
Comment (taburet) [inline hazelcast/src/main/java/com/hazelcast/monitor/impl/LocalMapStatsImpl.java]: Probably it also makes sense to rename the labels in the `toString` representation. It's unlikely that someone is relying on them.

**Comment by mmedenjak** [comment]
run-lab-run

**Comment by mmedenjak** [comment]
run-lab-run

**Pull request closed without merging**